resources/user-agents/browsers/chromium/chromium-55-on.json

```diff
@@ -65,6 +66,14 @@
           ]
         },
         {
+          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* Chromium/#MAJORVER#.*Safari/*",
```

Comment:
I only added this here since it's the only case I saw, though, I should add it to the generic division at least.